src/components/Wrapper.js

```diff
@@ -16,15 +15,14 @@ import {
 } from 'ramda';
 import { DesktopOnly, MobileOnly } from './MediaQuery';
 import MenuButtons from './MenuButtons';
-import Spacer from './Spacer';
 
 const drawerWidth = 142;
 const headerHeight = 50;
```

Comment:
Can just remove this as well and put the value directly on line 24 since it will be the only place it's used.